src/proto/FunctionRpc.proto

```diff
@@ -69,6 +70,9 @@ message StreamingMessage {
     FunctionEnvironmentReloadRequest function_environment_reload_request = 25;
 
     FunctionEnvironmentReloadResponse function_environment_reload_response = 26;
+	
+	//Used to send the dependency management request to the language worker
```

Comment:
It is optional for a language worker to implement a new message type. Functions host needs to look for capabilities published by a language worker before sending this request.